src/config/ssd.md

```diff
@@ -0,0 +1,147 @@
+# Solid State Drives
+
+Post installation, you will need to enable TRIM for solid state drives. You can
+check which devices allow TRIM by running:
+
+```
+$ lsblk --discard
+```
+
+If the DISC-GRAN (discard granularity) and DISC-MAX (discard maximum bytes)
+columns are non-zero, that means the block device has TRIM support. If your
+solid state drive partition does not show TRIM support, please verify that you
+chose a file system with TRIM support (ext4, Btrfs, F2FS, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. Note that F2FS
+requires kernel 4.19 or above to support TRIM. For example, if your / directory
+is on an SSD:
```

Comment:
The F2FS thing is still a bit out of place. Why not put it together with the list of file systems that support TRIM?

Reply:
> (ext4, Btrfs, F2FS, etc.). Note that F2FS requires kernel 4.19 or above to support TRIM.